src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -1085,6 +1085,10 @@ EOD;
                 $dhcpdconf .= "    filename \"{$dhcpifconf['filename64']}\";\n";
                 $dhcpdconf .= "  } else if option arch = 00:09 {\n";
                 $dhcpdconf .= "    filename \"{$dhcpifconf['filename64']}\";\n";
+                $dhcpdconf .= "  } else if option arch = 00:0a {\n";
+                $dhcpdconf .= "    filename \"{$dhcpifconf['filename32arm'] || $dhcpifconf['filename32']}\";\n";
```

Comment:
shouldn't this be a `??` and outside the string block?

Reply:
The last time I worked on PHP code, 5.3 was the latest version, so please forgive my rustyness.